java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -369,11 +371,6 @@ public void mset_returns_success() {
 
         // exercise
         CompletableFuture<String> response = service.mset(keyValueMap);
-        String payload = response.get();
```

Comment:
why? revert?

Reply:
OOPS! a rebasing issue, fixed in df50a9ba
thanks